x-pack/plugins/observability/public/rules/register_observability_rule_types.ts

```diff
@@ -125,7 +125,7 @@ export const registerObservabilityRuleTypes = (
       alertDetailsAppSection: lazy(
         () => import('../components/custom_threshold/components/alert_details_app_section')
       ),
-      priority: 110,
+      priority: 5,
```

Comment:
Priority is according to description of https://github.com/elastic/kibana/issues/166136